### PR TITLE
Promoted labelFingerprint field to GA

### PR DESCRIPTION
**Pull request by melinath**
Fixed https://github.com/hashicorp/terraform-provider-google/issues/18739.

This required writing a new Update test, so I implemented a PlanCheck to verify there was no deletion as described in https://github.com/hashicorp/terraform-provider-google/issues/12428#issuecomment-1548573984 The first commit intentionally forces recreate of the resource to show the PlanCheck works.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
compute: added `label_fingerprint` field to `google_compute_global_address` resource (ga)
```

```release-note:bug
compute: fixed bug where the `labels` field could not be updated on `google_compute_global_address` (ga)
```

### CODE REVIEW

File: mmv1/third_party/terraform/acctest/resource_test_utils.go
Comment (trodge) [inline mmv1/third_party/terraform/acctest/resource_test_utils.go:1]: Has this file been run through gofmt?

Comment (melinath) [reply mmv1/third_party/terraform/acctest/resource_test_utils.go:1]: It has now 😂  I'm waiting for VCR to finish before pushing new commits.

**Comment by melinath** [comment]
I see expected error in the VCR tests: `expected no deletion of resources, but google_compute_global_address.foobar has planned deletion`

**Review by trodge — APPROVED** [review]

**Comment by melinath** [comment]
remaining test failures are unrelated

**Comment by melinath** [comment]
opened https://github.com/hashicorp/terraform-provider-google/issues/19199 to track the failures.

**Review by c2thorn — APPROVED** [review]

**Pull request merged**